android-components/components/support/remotesettings/src/test/java/mozilla/components/support/remotesettings/RemoteSettingsClientTest.kt

```diff
@@ -0,0 +1,121 @@
+package mozilla.components.support.remotesettings
+
+import kotlinx.coroutines.runBlocking
+import mozilla.appservices.remotesettings.RemoteSettings
+import mozilla.appservices.remotesettings.RemoteSettingsRecord
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.robolectric.RobolectricTestRunner
+import java.io.File
+import java.io.FileReader
+import java.io.IOException
+
+@RunWith(RobolectricTestRunner::class)
+class RemoteSettingsClientTest {
+
+    private lateinit var mockRemoteSettings: RemoteSettings
+    private lateinit var mockFile: File
+    private lateinit var mockFileReader: FileReader
+    private lateinit var mockJsonObject: JSONObject
+    private lateinit var remoteSettingsClient: RemoteSettingsClient
+
+    @Before
+    fun setUp() {
+        mockFile = mock(File::class.java)
+        mockFileReader = mock(FileReader::class.java)
+        mockJsonObject = mock(JSONObject::class.java)
+        mockRemoteSettings = mock(RemoteSettings::class.java)
```

Comment:
Since the type of these mocks is declared above, you shouldn't need the explicit class arguments
```suggestion
        mockFile = mock()
        mockFileReader = mock()
        mockJsonObject = mock()
        mockRemoteSettings = mock()
```